pkg/dtest/lock.go

```diff
@@ -62,5 +64,6 @@ func WithNamedMachineLock(name string, body func()) {
 		file.Close()
 	}()
 
+	fmt.Printf("Acquiring machine lock %q took %.2f seconds\n", name, time.Since(lockAcquireStart).Seconds())
```

Comment:
Did you mean to leave this in?

Reply:
This `Printf` and the associated `grep` pipeline in the CI YAML are in the last commit so they can be dropped easily. I'd prefer to leave them in for now, until we improve the lock situation (e.g., via #179).